src/main/resources/assets/js/application/services/tdpInvestAuthService.js

```diff
@@ -32,7 +37,7 @@ define(['angular','application/tdpInvestModule', 'application/services/tdpInvest
         service.clearCredentials = function() {
             $rootScope.globals = {};
             $cookieStore.remove('globals');
-            $http.defaults.headers.common.Authorization = 'Basic ';
+            $http.defaults.headers.common.Authorization = '';
```

Comment:
Because with 'Basic ' links don't work on Safari - maybe we can implement a better solution.